.env

```diff
@@ -4,12 +4,13 @@ MONGO_URL=mongodb://0.0.0.0:27017/mua-system-admins
 SECRETS=verylongsecret1 verylongsecret2
 
 APP_URL=http://admins.emailfox.link/
-STATIC_SERVER_URL=http://localhost:10006/
-STATIC_SERVER_PORT=10006
+TEST_STATIC_SERVER_URL=http://localhost:10006/
```

Comment:
please don't use two separate env variables containing the same info (in this case it's the port)